### ISSUE TITLE
websocket server send text error 1002

### ISSUE THREAD

**Issue by baichangda** [helidon-io/helidon#7986]
## Environment Details
 implementation group: 'io.helidon.webserver', name: 'helidon-webserver', version: '4.0.0'
 implementation group: 'io.helidon.webserver', name: 'helidon-webserver-static-content', version: '4.0.0'
 implementation group: 'io.helidon.webserver', name: 'helidon-webserver-websocket', version: '4.0.0'

----------

## Problem Description
helidon se websocket server send text to client in onOpen function produce client error 1002

## Steps to reproduce
websocket server code like this 
![image](https://github.com/helidon-io/helidon/assets/24662536/fab359b8-29d8-4aa9-b783-852eaf270e6f)
webcoekt client code like this
![image](https://github.com/helidon-io/helidon/assets/24662536/b555d1a0-8294-41ea-a6e8-cf69d34721b3)

1、js websocket connect to helidon websocket server succeed
2、in onOpen(...) callback send msg to client、send function is 'WsSession send(String text, boolean last)'
3、then js websocket client error
![image](https://github.com/helidon-io/helidon/assets/24662536/1da2b648-a085-48ec-a5a2-f3dc8b8cb176)

**Comment by spericas** [comment]
@baichangda Seems like you're not showing the call to `send(String, boolean)` in your `onOpen`, right? Do you have sample code that matches your problem description? Is that in your `WsSession` class?

**Comment by baichangda** [comment]
@spericas there is call stack image
![image](https://github.com/helidon-io/helidon/assets/24662536/fb1457ae-5365-4259-b2fc-ffd56f097345)
![image](https://github.com/helidon-io/helidon/assets/24662536/9c48e512-ec59-4393-8c52-4d1f1be4aacd)
![image](https://github.com/helidon-io/helidon/assets/24662536/013b8160-5a59-4419-9f82-52da9b19f73a)
![image](https://github.com/helidon-io/helidon/assets/24662536/dc827160-f160-4080-9654-403b99bb792c)

**Comment by spericas** [comment]
@baichangda I'm not able to reproduce your problem using Helidon 4.0.0 and Chrome 119.0.6045.123. Please find a simple WS app attached that you can build and run as a server. For the client part, I'm just pasting some JavaScript code into the browser's console:

```
ws = new WebSocket("ws://localhost:8080/ws")
ws.onopen = function (evt) {
    console.log("onopen " + evt.type);
}
ws.onmessage = function (evt) {
    console.log("onmessage " + evt.data)
    ws.send("Hello to you")
}
ws.onclose = function (evt) {
    console.log("onclose " + evt.code)
}
```

The server and client outputs are all as expected, without any errors. Have your tried other clients? Perhaps you can use this simple app and modify it to reproduce the issue.

[quickstart-se.zip](https://github.com/helidon-io/helidon/files/13366723/quickstart-se.zip)

**Comment by baichangda** [comment]
@spericas 
i try send simple string, there is no problem
but complex string 、for example json like this、ws client will get error、you can try it
`"{\"@class\":\"com.bcd.share.support_parser.impl.gb32960.data.VehicleRunData\",\"collectTime\":1686211816000,\"vehicleBaseData\":{\"vehicleStatus\":1,\"chargeStatus\":3,\"runMode\":1,\"vehicleSpeed\":6.4,\"totalMileage\":85.0,\"totalVoltage\":388.0,\"totalCurrent\":11.299988,\"soc\":28,\"dcStatus\":1,\"gearPosition\":46,\"resistance\":8192,\"pedalVal\":0,\"pedalStatus\":0},\"vehicleMotorData\":{\"num\":1,\"content\":[{\"no\":1,\"status\":1,\"controllerTemperature\":49,\"rotateSpeed\":443,\"rotateRectangle\":1.5,\"temperature\":34,\"inputVoltage\":389.0,\"current\":0.0}]},\"vehiclePositionData\":{\"status\":0,\"lng\":121.193701,\"lat\":31.28577},\"vehicleLimitValueData\":{\"maxVoltageSystemNo\":1,\"maxVoltageCode\":9,\"maxVoltage\":3.611,\"minVoltageSystemNo\":1,\"minVoltageCode\":55,\"minVoltage\":3.604,\"maxTemperatureSystemNo\":1,\"maxTemperatureNo\":1,\"maxTemperature\":29,\"minTemperatureSystemNo\":1,\"minTemperatureNo\":4,\"minTemperature\":28},\"vehicleAlarmData\":{\"maxAlarmLevel\":3,\"alarmFlag\":136071,\"chargeBadNum\":0,\"chargeBadCodes\":[],\"driverBadNum\":0,\"driverBadCodes\":[],\"engineBadNum\":0,\"engineBadCodes\":[],\"otherBadNum\":0,\"otherBadCodes\":[]},\"vehicleStorageVoltageData\":{\"num\":1,\"content\":[{\"no\":1,\"voltage\":388.0,\"current\":11.299988,\"total\":108,\"frameNo\":1,\"frameTotal\":108,\"singleVoltage\":[3.608,3.609,3.61,3.609,3.609,3.608,3.608,3.61,3.611,3.608,3.609,3.61,3.608,3.608,3.609,3.61,3.61,3.609,3.608,3.61,3.608,3.61,3.61,3.608,3.607,3.609,3.607,3.609,3.607,3.609,3.611,3.609,3.609,3.609,3.608,3.608,3.607,3.607,3.608,3.607,3.607,3.607,3.609,3.607,3.608,3.607,3.609,3.607,3.607,3.607,3.608,3.608,3.609,3.609,3.604,3.608,3.608,3.607,3.607,3.605,3.606,3.606,3.608,3.609,3.607,3.608,3.607,3.608,3.607,3.608,3.607,3.606,3.609,3.605,3.608,3.606,3.608,3.607,3.606,3.606,3.607,3.605,3.607,3.607,3.604,3.607,3.606,3.606,3.607,3.607,3.607,3.607,3.606,3.607,3.606,3.607,3.604,3.607,3.607,3.606,3.606,3.607,3.607,3.607,3.606,3.606,3.606,3.606]}]},\"vehicleStorageTemperatureData\":{\"num\":1,\"content\":[{\"no\":1,\"num\":12,\"temperatures\":[29,29,29,28,29,28,28,28,29,28,29,28]}]}}"`

**Comment by spericas** [comment]
Issue has been fixed, will be available in release 4.0.1.
